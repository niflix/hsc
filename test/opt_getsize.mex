opt_getsize.hsc (10,33): Warning 68: filetype of `test.ilbm' not recognised
