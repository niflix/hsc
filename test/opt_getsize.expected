<html><head><title>Test Image sizes</title></head><body>

<img src="test.gif" alt="test" width="320" height="256">
<img src="test.png" alt="test" width="320" height="256">
<img src="test.jfif" alt="test" width="320" height="256">

<img src="test_eof.gif" alt="test" width="56" height="46">

<img src="test.ilbm" alt="test"> 
</body></html>
